views/install.sh.erb

```diff
@@ -439,8 +447,13 @@ install_file() {
   echo "Installing $project $version"
   case "$1" in
     "rpm")
-      echo "installing with rpm..."
-      rpm -Uvh --oldpackage --replacepkgs "$2"
+      if test "x$platform" = "xnexus" || test "x$platform" = "xios_xr"; then
+        echo "installing with yum..."
+        yum install -yv "$2"
```

Comment:
that smells a bit incorrect.  if we're correctly building an omnibus binary we shouldn't have any deps outside of the base operating systems so shouldn't need to depsolve (which is why i assume you're doing this?  otherwise you have a yum binary with no rpm binary, which is very odd...)